### PR TITLE
ZKUI-369: identity-group-names-become-long

### PR DESCRIPTION
**Pull request by hervedombya**
Screenshot 2023-07-10 at 09 59 22

Screenshot 2023-07-10 at 16 30 37

Screenshot 2023-07-10 at 16 31 21

### CODE REVIEW

File: src/react/endpoint/EndpointList.tsx [hunk 1]
```diff
@@ -64,8 +64,13 @@ function EndpointList({ endpoints, locations }: Props) {
         },
         Cell({ value: hostName }: { value: Hostname }) {
           return (
-            <div style={{ display: 'flex' }}>
-              <span style={{ paddingRight: spacing.sp14 }}>{hostName}</span>
+            <div style={{ display: 'flex', width: `calc(100% - 100px)` }}>
```
Comment (JBWatenbergScality) [inline src/react/endpoint/EndpointList.tsx]: Why `-100px` ?

Comment (hervedombya) [reply src/react/endpoint/EndpointList.tsx]: to align icons regardless of text length

Screenshot 2023-07-10 at 17 50 47

Comment (JBWatenbergScality) [reply src/react/endpoint/EndpointList.tsx]: Can't we replace it by a `<Wrap />` ?

**Review by JBWatenbergScality — APPROVED** [review]
LGTM !

**Pull request merged**